EKF/mag_fusion.cpp

```diff
@@ -173,6 +173,10 @@ void Ekf::fuseMag()
 	// before they are used to constrain heading drift
 	const bool update_all_states = ((_imu_sample_delayed.time_us - _flt_mag_align_start_time) > (uint64_t)5e6);
 
+	_fault_status.flags.bad_mag_x = false;
```

Comment:
Nice, that's something really hard to see with those huge function cluttered with auto-generated code...